values.yaml

```
@@ -1311,9 +1311,12 @@ concourse:
     externalGardenUrl:
 
     garden:
-      ## Path to the 'gdn' executable (or leave as 'gdn' to find it in $PATH)
+      ## Path to the executable to run the garden backend ('gdn' or
+      ## 'containerd').
       ##
-      bin: gdn
+      ## If not specified, the right binary will be search in $PATH.
```

```suggestion
      ## If not specified, Concourse will search $PATH for the right binary.
```
flows a little bit better